### ISSUE TITLE
[@typescript-eslint/member-ordering] Member should be declared before all private instance field definitions

### ISSUE THREAD

**Issue by woteska** [typescript-eslint/typescript-eslint#3146]
- [x] I have tried restarting my IDE and the issue persists.
- [x] I have updated to the latest version of the packages.
- [x] I have [read the FAQ](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/FAQ.md) and my problem is not listed.

**Repro**

```TS
@Injectable({
  providedIn: 'root'
})
export class SomeService {
  private readonly someProtectedData$ = new BehaviorSubject<number>(0);
  readonly someData$ = this.someProtectedData$.asObservable();
}
```

**Expected Result**

No '@typescript-eslint/member-ordering' error, because the public one is depending on a private one.

**Actual Result**

ESLint: Member someData$ should be declared before all private instance field definitions. (@typescript-eslint/member-ordering)

**Additional Info**

See above and below.

**Versions**

ESLinter was added by 'ng add @angular-eslint/schematics' (following this step list: https://github.com/angular-eslint/angular-eslint#migrating-an-angular-cli-project-from-codelyzer-and-tslint) to a CLI project generated by @angular/cli@11.2.0.

    "eslint": "^7.6.0",
    "eslint-plugin-import": "2.22.1",
    "eslint-plugin-jsdoc": "30.7.6",
    "eslint-plugin-prefer-arrow": "1.2.2",
    "@angular-eslint/builder": "1.2.0",
    "@angular-eslint/eslint-plugin": "1.2.0",
    "@angular-eslint/eslint-plugin-template": "1.2.0",
    "@angular-eslint/schematics": "1.2.0",
    "@angular-eslint/template-parser": "1.2.0",
    "@typescript-eslint/eslint-plugin": "4.3.0",
    "@typescript-eslint/parser": "4.3.0"
    
    Typescript: 4.1.5
    npm: 6.14.11
    Node.js: 14.15.4

### How to resolve this, if it is not a 'bug'?

**Comment by bradzacher** [comment]
Duplicate of https://github.com/typescript-eslint/typescript-eslint/issues/2882
Please use the search
